src/trie.js

```diff
@@ -100,8 +100,10 @@ var marexandre;
             // Check if next character exists in children, and if does dive deeper
             if (copy[j + 1]) {
               var exists2 = tmpTrie.children.hasOwnProperty(copy[j + 1].toString());
-              if (tmpTrie.is_end && !exists2) {
+              if (tmpTrie.is_end) {
```

Comment:
Ah, I see. `is_end` means "is end of word", not "is end of sentence".

I would still inline `exists2` in the `if`.